server/lib/nl/common/bad_words.py

```diff
@@ -69,8 +69,14 @@ class BannedWords:
 
 #
 # Loads a list of bad words from a text file.
+# If the load argument is False (True by default),
+# then an empty BannedWords object will be returned.
+# It will only be True for Custom DCs at this time.
 #
-def load_bad_words() -> BannedWords:
+def load_bad_words(load: bool = True) -> BannedWords:
+  if not load:
+    return BannedWords(entries={})
+
```

Comment:
Instead of this additional bool arg, maybe we can create an empty entry explicitly if CUSTOM DC?

Also, in `is_safe`, return True if `not bad_words.entries`?

Reply:
> Also, in is_safe, return True if not bad_words.entries?

Done.

> create an empty entry explicitly

Want to keep it encapsulated in this method, in case we want to load them differently?

Reply:
> > create an empty entry explicitly
> 
> Want to keep it encapsulated in this method, in case we want to load them differently?

nit: It just seems odd to call load_bad_words(load=False)?  Feels slightly better to leave it empty, or call `empty_bad_words()` in the caller.

Reply:
Ok. Done.